### PR TITLE
[15.0] Migrating partner contact tradename module

### PR DESCRIPTION
**Pull request by patovala**

### CODE REVIEW

File: partner_contact_tradename/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    tradename = fields.Char(
+        size=300,
+    )
+
+    @api.model
+    def name_search(self, name="", args=None, operator="ilike", limit=100):
+        args = args or []
+        try:
+            recs = self.search(
```
Comment (cormaza) [inline partner_contact_tradename/models/res_partner.py:19]: Should be good super call for other implementations of this feature or another modules

**Review by cormaza — CHANGES_REQUESTED** [review]
Please check super call, because this class is used very much with implementations on name_search

File: partner_contact_tradename/models/res_partner.py (hunk 1 above)
Comment (patovala) [reply partner_contact_tradename/models/res_partner.py:19]: Yes, will do the fix asap, thanks

**Comment by cormaza** [comment]
Can be good idea check this implementation:

https://github.com/OCA/l10n-spain/blob/96ce681c8ff0d0c5e53aea2e6734acb8f87dd73f/l10n_es_partner/models/res_partner.py#L39-L57

**Pull request closed without merging**